app/views/background_jobs/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="modal-body">
+  <h1>Background Job</h1>
+  <hr/>
+  <div class="container-fluid">
+    <h5><strong><%= @background_job.created_at.strftime('%B %d, %Y at %l:%M %p') %></strong></h5>
+  </div>
```

Comment:
If the current_user is admin, show the username of the user that created the background job